No tumor in reference data (#271)
AlexsLemonade/scpca-docs: docs/processing_information.md

During annotation, we additionally include an `"other"` cell type that does not express any of these marker genes.
As a consequence, cells which `CellAssign` cannot confidently annotate from the full marker gene list are labeled as `"other"`.

> Please be aware that all cell type annotation reference datasets include only normal (not tumor) tissue.

Reviewer: ```suggestion
Please be aware that all cell type annotation reference datasets are derived from normal (not tumor) tissue.
```
How about this?

Author: good with me!
